editor/editor_asset_installer.h

```diff
@@ -33,25 +33,49 @@
 
 #include "scene/gui/dialogs.h"
 #include "scene/gui/tree.h"
+
+class CheckBox;
+class Label;
```

Comment:
Why was this added if Label was already used without problems?

Reply:
Just a precaution because these are used without a problem implicitly. I'm not sure dialogs and tree should guarantee these to be included (likely could benefit from forward declarations as well).

Reply:
There are many more classes included explicitly.